versions/3.1.1.md

```diff
@@ -248,18 +248,28 @@ OAS < 3.1 | OAS 3.1 | Comments
 ### <a name="richText"></a>Rich Text Formatting
 Throughout the specification `description` fields are noted as supporting CommonMark markdown formatting.
 Where OpenAPI tooling renders rich text it MUST support, at a minimum, markdown syntax as described by [CommonMark 0.27](https://spec.commonmark.org/0.27/). Tooling MAY choose to ignore some CommonMark features to address security concerns. 
+Hyperlinks in CommonMark text are considered to be [#relativeReferencesURI](API description URIs).
```

Comment:
This link looks wrong. Should it be something like
```suggestion
Hyperlinks in CommonMark text are considered to be [relative references](#relativeReferencesURI).
```

Reply:
@ralfhandl thanks for bringing this up, I had to think about it a bit because it's not clear to me how this interacts with rendering the Markdown.  I'm going to take this part out and continue the discussion in issue #1701, which I will take off of the "fixed" list for this PR.